src/proxy/topic/ReliableTopicListenerRunner.ts

```diff
@@ -65,9 +69,13 @@ export class ReliableTopicListenerRunner<E> {
 
                 const nextSeq = result.getNextSequenceToReadFrom().toNumber();
                 const lostCount = nextSeq - result.getReadCount() - this.sequenceNumber;
+                // If messages were lost, behave as a non-loss tolerant listener
                 if (lostCount !== 0) {
-                    this.logger.trace('ReliableTopicListenerRunner', 'Listener of topic: ' + this.proxy.getName()
-                        + ' lost ' + lostCount + ' messages');
+                    this.logger.warn('ReliableTopicListenerRunner', 'Terminating listener of topic: '
+                        + this.proxy.getName() + '. Reason: The listener was too slow lost or the retention'
```

Comment:
Minor: `Reason: The listener was too slow lost or the retention...` should be `Reason: The listener was too slow or the retention...` (extra `lost`)

Reply:
Fixed in ab19d9c259760e62bccbcb020cf913f3e78572f8